packages/template-retail-react-app/app/commerce-api/auth.js

```diff
@@ -139,23 +140,44 @@ class Auth {
         this._storage.set(oidStorageKey, oid)
     }
 
+    get isTokenValid() {
+        return (
+            isTokenValid(this.authToken) &&
```

Comment:
If the old `isTokenValid` just checks for expiration, I'd suggest we rename the old function to `isTokenExpired`